docs/topics/migration-paths.adoc

```diff
@@ -4,15 +4,13 @@
 
 :_content-type: CONCEPT
 [id="migration-paths_{context}"]
-= Supported migration paths
+= Supported {ProductName} migration paths
 
-The {ProductName} ({ProductShortName}) supports migrations from third-party enterprise application servers, such as Oracle WebLogic Server, to JBoss Enterprise Application Platform (JBoss EAP) and upgrades to the latest release of JBoss EAP.
+The {ProductName} ({ProductShortName}) supports migrations from third-party enterprise application servers, such as Oracle WebLogic Server, to JBoss Enterprise Application Platform (JBoss EAP), and upgrades to the latest release of JBoss EAP.
```

Comment:
It's really hard to follow this list. Recommend reformatting as bullet points.

Reply:
Nice! Will incorporate in the next commit